src/sentry/static/sentry/app/components/events/contextSummary/contextSummaryGeneric.tsx

```diff
@@ -40,7 +40,7 @@ const ContextSummaryGeneric = ({data, unknownTitle}: Props) => {
     );
   };
 
-  const className = generateClassName(data.name);
+  const className = data.name ? generateClassName(data.name) : '';
```

Comment:
You have this in a few places. Should `generateClassName()` be handle this situation? It could save future us.